### PR TITLE
 Operation labelled as flow

### LINKED ISSUE #531: Operation labelled as flow
When running an operation directly from the CLI (without a calling flow) the text printed to the screen still says "Flow outputs:" and "Flow xxx finished with result :".

It's a little misleading.

### PR DESCRIPTION
**Pull request by gituser173**
Fixes #531 
Signed-off-by: Claudiu Ban claudiu-laurian.ban@hpe.com

### CODE REVIEW

**Comment by gituser173** [comment]
@Bonczidai please review

File: cloudslang-cli/src/main/java/io/cloudslang/lang/cli/services/SyncTriggerEventListener.java [hunk 1]
```diff
@@ -107,7 +110,12 @@ else if(data.containsKey(LanguageEventData.OUTPUTS)
                         && data.containsKey(LanguageEventData.PATH)
                         && data.get(LanguageEventData.PATH).equals(EXEC_START_PATH)) {
                     Map<String, Serializable> outputs = extractNotEmptyOutputs(data);
-                    printWithColor(Ansi.Color.WHITE, "\nFlow outputs:");
+                    if (data.containsKey(ScoreLangConstants.EXECUTABLE_TYPE) &&
+                            ExecutableType.OPERATION.equals(data.get(ScoreLangConstants.EXECUTABLE_TYPE))) {
```
Comment (Bonczidai) [inline cloudslang-cli/src/main/java/io/cloudslang/lang/cli/services/SyncTriggerEventListener.java]: you can remove `data.containsKey(ScoreLangConstants.EXECUTABLE_TYPE) &&` from the condition

File: cloudslang-entities/src/main/java/io/cloudslang/lang/entities/ScoreLangConstants.java [hunk 2]
```diff
@@ -18,6 +18,7 @@
     String RUN_ENV = "runEnv";
     String HOOKS = "hooks";
     String NODE_NAME_KEY = "nodeName";
+    String EXECUTABLE_TYPE = "executableType";
```
Comment (Bonczidai) [inline cloudslang-entities/src/main/java/io/cloudslang/lang/entities/ScoreLangConstants.java]: IMHO the correct place for `EXECUTABLE_TYPE` type would be in `LnaguageEventData`

File: cloudslang-entities/src/main/java/io/cloudslang/lang/entities/ExecutableType.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
+package io.cloudslang.lang.entities;
+
+/**
+ * User: bancl
+ * Date: 1/6/2016
+ */
+public enum ExecutableType {
```
Comment (Bonczidai) [inline cloudslang-entities/src/main/java/io/cloudslang/lang/entities/ExecutableType.java]: you should put this enum in `LnaguageEventData` similar to `StepType`

**Comment by Bonczidai** [comment]
very nice :+1: 

please handle comments

File: cloudslang-entities/src/main/java/io/cloudslang/lang/entities/ScoreLangConstants.java (hunk 2 above)
Comment (gituser173) [inline cloudslang-entities/src/main/java/io/cloudslang/lang/entities/ScoreLangConstants.java]: we need to add a dependency to cloudslang-runtime in the cloudslang-compiler pom to do that.

File: cloudslang-entities/src/main/java/io/cloudslang/lang/entities/ExecutableType.java (hunk 3 above)
Comment (gituser173) [inline cloudslang-entities/src/main/java/io/cloudslang/lang/entities/ExecutableType.java]: we need to add a dependency to cloudslang-runtime in the cloudslang-compiler pom to do that.

Comment (Bonczidai) [inline cloudslang-entities/src/main/java/io/cloudslang/lang/entities/ExecutableType.java]: right, so let's stay with the current approach

**Comment by gituser173** [comment]
capture

**Comment by gituser173** [comment]
capture
capture2

**Comment by Bonczidai** [comment]
:ok_hand:

**Pull request merged**